packages/core/src/stylable-transformer.ts

```diff
@@ -230,6 +239,12 @@ export class StylableTransformer {
                     atRule,
                     resolved: layerResolve,
                 });
+            } else if (name === 'container') {
```

Comment:
I wish that the checks would inline to the hook and see a flat list here, maybe we want to open issue for it.

Reply:
I will open an issue the to refactor and unify the code in this area